src/canmatrix/log.py

```diff
@@ -27,8 +27,8 @@
 import logging
 
 
-def setup_logger(name):
-    """Setup a project wide logger singleton"""
+def setup_logger(name='canmatrix'):
+    """Setup the module logger for canmatrix module. Return logger instance for possible further setting."""
```

Comment:
```suggestion
    """Setup the logger.  By default this is the root logger. Return logger instance for possible further setting and use."""
```

If above is changed...